README.md

```diff
@@ -82,7 +89,12 @@ theme
 missing-translations
 ├── message_ar.properties    Arabic
 ├── message_da.properties    Danish
-├── message_de.properties    German
+├── message_de_AT.properties    German (Austria)
```

Comment:
These will only be present if there are any missing translations (you can see that `cz` is all done, so is not present). Please remove the German entries.

Other than that, looks good to go.

Reply:
These lines under `missing-translations`...

Reply:
included in the de_IT commit.